db/migrate/20230408145819_add_guest_comment_off_to_admin_settings.rb

```diff
@@ -0,0 +1,5 @@
+class AddGuestCommentOffToAdminSettings < ActiveRecord::Migration[6.1]
+  def change
+    add_column :admin_settings, :guest_comments_off, :boolean, default: false, null: false
```

Comment:
Since you're adding a new column here, I think it might be good to change the key used to cache the admin settings. That way we can keep the deploy steps simpler:
1. Run the migration.
2. Do a rolling restart of the servers and workers with the new code.

The alternative would be to add some extra steps to the deploy to prevent errors:
1. Make sure that all admins know not to update the admin settings during this period, and make sure that the deploy isn't going to overlap with 6:21 AM UTC, when the invitations code changes the admin settings.
2. Run the migration.
3. Open a copy of the rails console with the new code and run `AdminSetting.first.recache_settings`.
4. Do a rolling restart of the servers and workers with the new code.